src/main/scala/nl/knaw/dans/easy/ingest/EasyIngest.scala

```diff
@@ -127,6 +126,7 @@ object EasyIngest {
     for {
       foxml <- getFOXML(sdo)
       pid <- executeIngest(configDictionary(sdo.getName), foxml)
+      _ = log.info(s"ingested $pid $foxml")
```

Comment:
Doesn't this produce too much `info` logging? I mean, `$foxml` is a full xml file, right?